tests/e2e/notes.test.ts

```diff
@@ -0,0 +1,74 @@
+import { faker } from '@faker-js/faker'
+import { expect, test } from '../playwright-utils.ts'
+
+test('Users can create notes', async ({ login, page }) => {
+	const user = await login()
+	await page.goto(`/users/${user.username}/notes`)
+
+	const newNote = createNewNote()
+	await page.locator("//a[text()='+ New Note']").click()
```

Comment:
Let's swap from `.locator` to the testing-library inspired API playwright added a while ago. Check the recommended locators here: https://playwright.dev/docs/locators#quick-guide

Reply:
Thanks! Removed pretty much all `.locator` usage, except a couple of instances where I needed to verify if deletion works as intended.